pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -679,6 +679,18 @@ def compute_decomposition(wires):
         """
         return [qml.PhaseShift(np.pi / 2, wires=wires)]
 
+    def pow(self, z):
```

Comment:
how come pow is defined? 🤔

Reply:
Should we define the adjoint as `qml.PhaseShift(-np.pi/2, wires=op.wires)`? Like how we're simplifying powers of `S`?